### DISCUSSION TITLE
Welcome to dataclasses-json Discussions!

### DISCUSSION THREAD

**Discussion by lidatong** [lidatong/dataclasses-json#271] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by CodeYan01** [comment]
Hi, I haven't started using dataclasses yet, but I'm wondering if generating dataclasses from JSON schemas is within the scope of this project.
